app/components/anchor/table_component.rb

```diff
@@ -1,20 +1,21 @@
 module Anchor
   class TableComponent < Component
     renders_many :columns, "ColumnComponent"
-    renders_one :link
 
-    attr_reader :data, :sortable, :caption, :show_total
+    attr_reader :data, :sortable, :caption, :rows_link_to, :show_total
```

Comment:
(nitpick) Since it'll be run over each row. 
```suggestion
    attr_reader :data, :sortable, :caption, :row_link_to, :show_total
```